cypress/support/nodeManager.ts

```diff
@@ -24,6 +29,37 @@ export const connectTwoNodes = (
   );
 };
 
+interface createCommitOptions {
+  repositoryPath: string;
+  monorepoPath: string;
+  subject: string;
+  passphrase: string;
+  name?: string;
+  email?: string;
+}
+
+const credentialsHelper = (passphrase: string) =>
+  `'!f() { test "$1" = get && echo "password=${passphrase}"; }; f'`;
+
+export const createCommit = (options: createCommitOptions): void => {
+  cy.exec(
+    `set -euo pipefail
+export GIT_AUTHOR_NAME="${options.name || "John McPipefail"}"
+export GIT_AUTHOR_EMAIL="${options.email || "john@pipefail.com"}"
+export GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+export PATH=$PWD/proxy/target/release:$PATH
+cd ${options.repositoryPath}
+git commit --allow-empty -m "${options.subject}"
+git -c credential.helper=${credentialsHelper(options.passphrase)} push rad`,
+    {
+      env: {
+        RAD_HOME: options.monorepoPath,
```

Comment:
We are using two ways now to expose environment variables. Via `export`  in the script or adding them here. I’d prefer to just use one.

Reply:
`RAD_HOME` is not exactly the monorepo path. To avoid confusion I’d suggest to just call the property `radHome`, then it’s unambiguous what it is for.

Reply:
> We are using two ways now to expose environment variables. Via export in the script or adding them here. I’d prefer to just use one.

I'm not sure I follow, could you please elaborate?

> RAD_HOME is not exactly the monorepo path. To avoid confusion I’d suggest to just call the property radHome, then it’s unambiguous what it is for.

Good thinking. I like how everything now falls into place: 7463b20192904ac8811c12b29a751291c666c948.

Reply:
> We are using two ways now to expose environment variables. Via export in the script or adding them here. I’d prefer to just use one.

Thanks for clearing it up, makes sense to do it consistently: 14c09f337d7c54f194c8a4c232093332e41579ab.